docker-compose.yml

```diff
@@ -6,6 +6,7 @@ services:
     build:
       context: elasticsearch/
       args:
+        ELK_FLAVOUR: $ELK_FLAVOUR
```

Comment:
Maybe keep `ELK_FLAVOUR` below `ELK_VERSION` to keep the order of the env file.

Reply:
changed